src/xbot/common/controls/sensors/XGamepad.java

```diff
@@ -1,10 +1,33 @@
 package xbot.common.controls.sensors;
 
+
 public interface XGamepad {
 
-    public XJoystick getLeftStick();
-    public XJoystick getRightStick();
-    public XJoystick getDpad();
-    public XJoystick getLeftTrigger();
-    public XJoystick getRightTrigger();
+    //double
+    //Joysticks
+    double getXStickLeft();
```

Comment:
Logically, I think that it would make more sense to narrow scope in order. I'm thinking `getLeftStickX`. Same for others.